common/src/test/java/org/opensearch/ml/common/transport/load/MLLoadModelRequestTest.java

```diff
@@ -0,0 +1,122 @@
+         MLLoadModelRequest request = MLLoadModelRequest.builder().
+                 modelId("modelId").
+                 build();
+        assertNull(request.validate());
+    }
+
+    @Test
+    public void fromActionRequest_Success_WithMLUploadModelRequest() {
+        MLLoadModelRequest request = MLLoadModelRequest.builder().
+                modelId("modelId").
+                build();
+        assertSame(MLLoadModelRequest.fromActionRequest(request), request);
+    }
+
+    @Test
+    public void validate_Exception_NullModelId() {
+        MLLoadModelRequest request = MLLoadModelRequest.builder().
+                modelId(null).
+                modelNodeIds(new String[]{"modelNodeIds"}).
+                async(true).
+                dispatchTask(true).
+                build();
+        ActionRequestValidationException exception = request.validate();
+        assertEquals("Validation Failed: 1: ML model id can't be null;", exception.getMessage());
+    }
+
+    @Test
+    public void writeTo_Success() throws IOException {
+
+        MLLoadModelRequest request = mlLoadModelRequest;
+        BytesStreamOutput bytesStreamOutput = new BytesStreamOutput();
+        request.writeTo(bytesStreamOutput);
+        request = new MLLoadModelRequest(bytesStreamOutput.bytes().streamInput());
+
+        assertEquals("modelId", request.getModelId());
+        assertArrayEquals(new String[]{"modelNodeIds"}, request.getModelNodeIds());
+        assertTrue(request.isAsync());
+        assertTrue(request.isDispatchTask());
+    }
+
+    @Test(expected = UncheckedIOException.class)
+    public void fromActionRequest_IOException() {
+        ActionRequest actionRequest = new ActionRequest() {
+            @Override
+            public ActionRequestValidationException validate() {
+                return null;
+            }
+
+            @Override
+            public void writeTo(StreamOutput out) throws IOException {
+                throw new IOException("test");
+            }
+        };
+        MLLoadModelRequest.fromActionRequest(actionRequest);
+    }
+
+    @Test
+    public void fromActionRequest_Success_WithNonMLUploadModelRequest_ConfigInput() {
+        MLLoadModelRequest request = mlLoadModelRequest;
+        ActionRequest actionRequest = new ActionRequest() {
+            @Override
+            public ActionRequestValidationException validate() {
+                return null;
+            }
+
+            @Override
+            public void writeTo(StreamOutput out) throws IOException {
+                request.writeTo(out);
+            }
+        };
+        MLLoadModelRequest result = MLLoadModelRequest.fromActionRequest(actionRequest);
+        assertNotSame(result, request);
+        assertEquals(request.isAsync(), result.isAsync());
+        assertEquals(request.isDispatchTask(), result.isDispatchTask());
+    }
+
+
+/*
+    // No toXcontent Method?
+    @Test
+    public void testParse() throws Exception {
+    }
+ */
+}
```

Comment:
Can you add "extends ToXContentObject" for MLLoadModelRequest, create a MLLoadModelRequest instance, and use toXContent() to create builder, and verify the parser() result? Use this as reference https://github.com/opensearch-project/ml-commons/blob/2.x/plugin/src/test/java/org/opensearch/ml/profile/MLProfileInputTests.java#L50